accelerate-tensorflow-lite/package.yaml

```diff
@@ -56,17 +56,17 @@ library:
     -march=native
 
   extra-libraries:
-    - stdc++
-    - clog
-    - cpuinfo
-    - edgetpu
-    - farmhash
-    - fft2d_fftsg
-    - fft2d_fftsg2d
-    - flatbuffers
-    - pthreadpool
-    - ruy
-    - tensorflow-lite
-    - XNNPACK
+      stdc++
+      clog
+      cpuinfo
+      edgetpu
+      farmhash
+      tensorflow-lite
+      ruy
+      XNNPACK
+      flatbuffers
+      pthreadpool
+      fft2d_fftsg
+      fft2d_fftsg2d
```

Comment:
I think the recommendation these days is to also commit the corresponding change to the .cabal file (even though it's auto generated)?

Reply:
Oh, my bad. I'll add these to a commit right now!